src/scripts/docker-buildx.sh

```diff
@@ -37,10 +37,6 @@ if [ "${PARAM_SKIP_WHEN_TAGS_EXIST}" = "0" ] || [[ "${PARAM_SKIP_WHEN_TAGS_EXIST
     set -- "$@" --push
   fi
 
-  if [ -n "$PARAM_EXTRA_BUILD_ARGS" ]; then
-    set -- "$@" "${PARAM_EXTRA_BUILD_ARGS}"
-  fi
```

Comment:
we could also leave this block but drop only double-quotes. e.g.: `set -- "$@" ${PARAM_EXTRA_BUILD_ARGS}`